editoast/src/views/layers/mod.rs

```diff
@@ -80,17 +81,30 @@ async fn layer_view(
         return Err(LayersError::new_view_not_found(view_slug, layer).into());
     }
 
-    let tiles_url_pattern = format!(
-        "{root_url}/layers/tile/{layer_slug}/{view_slug}/{{z}}/{{x}}/{{y}}/?infra={infra}",
-        root_url = get_root_url()
-    );
+    let root_url = get_root_url();
+    let mut tiles_url_pattern = Url::parse(&root_url)?;
```

Comment:
We can directly add the params:
```suggestion
    let mut tiles_url_pattern = Url::parse_with_params(&root_url, [("infra", infra.to_string())])?;
```